remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -1,122 +1,117 @@
 #!/bin/bash
 
-install_driver()
+if [ $# -eq 1 ]
+then
+    AGENT_TYPE="$1"
+else
+    # the first argument is the Registration Code of PCoIP agent
+    REGISTRATION_CODE="$1"
+    # the second argument is the agent type
+    AGENT_TYPE="$2"
+    # the third argument is the VM name
+    VM_NAME="$3"
+    # the forth argument is the domain name
+    DOMAIN_NAME="$4"
+    # the fifith argument is the username
+    USERNAME="$5"
+    # the sixth argument is the password
+    PASSWORD="$6"
+    # the seventh argument is the domain group to join
+    GROUP="$7"
+    # the eighth argument is the agent installer channel
+    AGENT_CHANNEL="$8"
+    # the ninth argument is the userStorageAccountUri
+    STORAGEURI="$9"
+    # the tenth argument is the sumo collector ID
+    COLLECTOR_ID="${10}"
+    # the eleventh argument is the SaS Storage account token
+    SAS_TOKEN="${11}"
+    # the twelfth argument is the Computer OU string
+    OU="${12}"
+fi
+
+update_kernel_dkms() 
 {
-    # Install kernel-source/kernel-devel and gcc
-    sudo yum -y install kernel-source kernel-devel gcc
-    FILE_NAME="NVIDIA-Linux-x86_64-367.106-grid.run"
-    FILE_LOCATION="/root/$FILE_NAME"
-    # Download Driver first
-    echo "Downloading and Installing Nvidia driver"
-    sudo wget --retry-connrefused --tries=3 --waitretry=5 -O $FILE_LOCATION https://binarystore.blob.core.windows.net/thirdparty/nvidia/$FILE_NAME
-    exitCode=$?
-    if [ $exitCode -ne 0 ]
+  sudo yum -y update
+
+  sudo yum -y install kernel-devel
+
+  sudo rpm -Uvh --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+  sudo yum -y install dkms
+}
+
+# need to reboot after install
+disable_nouveal()
```

Comment:
I think there is a typo here... This should be `disable_nouveau()`.

Reply:
will change